### PR TITLE
Re former

### PR DESCRIPTION
**Pull request by mricanho**
In this project, I tried multiple ways to build a form that can send parameters to a controller so that it can build an instance of a model with those parameters.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+ReFormer::Application.routes.draw do
+  resources :users, :only => [:new, :create, :edit, :update]
```
Comment (jubaan) [inline config/routes.rb]: - [x] OPTIONAL: `only: [:new, :create, :edit, :update]`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
```
Comment (jubaan) [inline app/controllers/users_controller.rb:23]: - [x] should check if the user is saved, not updated

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <%= label_tag(:email, "email:") %>
+    <%= text_field_tag(:email) %><br>
+
+    <%= label_tag(:password, "password:") %>
+    <%= password_field_tag(:password) %><br>
+
+    <%= submit_tag("Submit") %>
+
+  <% end %> -->
+  <<h1>New User View</h1>
```
Comment (jubaan) [inline app/views/users/new.html.erb]: - [x] has an extra `<`

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    <input id="email" name="user[email]" type="text"><br>
+
+    <label for="password">Password:</label>
+    <input id="password" name="user[password]" type="password"><br>
+
+    <input type="submit" value="Submit"> 
+
+  </form>%>
+  
+  <!-- <%= form_tag("/users", method: "post") do %>
```
Comment (jubaan) [inline app/views/users/new.html.erb]: - [x] Use `form_tag(users_path)` without the `method: "post"`. This should work as well.

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New</title>
+</head>
+<body>
+  <%# <h1>Hey!</h1> %>
+  <%# <form accept-charset="UTF-8" action="/users" method="post">
```
Comment (jubaan) [inline app/views/users/new.html.erb]: - [x] Use `action="user_path"`

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @mricanho. Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] In the README is missing some setup steps, please add a step to change directory into the just cloned project and another one before running `rails db:migrate` to run `bundle install`
- [x] MIT License link in README is broken.
- [x] See inline comments for the rest of the observations:

### [OPTIONAL]  :yellow_circle: 
- [x] The preview screenshot would be better if it shows the view of the code.
- [x] Use the new `hash` syntax in routes. See inline comments

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the development branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**